ex1e.sh

```diff
@@ -0,0 +1,32 @@
+#1
+touch File{A,B,C,D,E,F}.txt

```

Comment:
This works, but you were expected to use the ".." subtitution like this: touch File{A..F}.txt